pages/bitcoin-core-extensions/electrs.en.mdx

```diff
@@ -103,64 +269,64 @@ RestartSec=30
 WantedBy=multi-user.target
 ```
 
-Sla het op met `Ctrl + X` en bevestigen met `Y`.
+Save it with `Ctrl + X` and confirm with `Y`.
 
-Enable en start de service met deze twee commando's:
+Enable and start the service with these two commands:
 
 ```bash
 sudo systemctl enable electrs
 
 sudo systemctl start electrs
 ```
 
-Met `sudo systemctl status electrs` kun je zien of alles goed draait en krijg je een output die lijkt op het onderstaande.
+With `sudo systemctl status electrs` you can see if everything is running properly and get an output similar to the one below.
 
 ```bash
-● electrs.service - electrs
+- electrs.service - electrs
      Loaded: loaded (/etc/systemd/system/electrs.service; enabled; vendor preset: enabled)
      Active: active (running) since Fri 2022-01-14 20:36:59 UTC; 35s ago
    Main PID: 67163 (electrs)
       Tasks: 20 (limit: 4203)
      CGroup: /system.slice/electrs.service
-             └─67163 /home/ubuntu/electrs/target/release/electrs
+             67163 /home/ubuntu/electrs/target/release/electrs
 ```
 
-## Updaten
+## Updating
 
-Ga naar de applicatie directory.
+Go to the application directory.
 
 ```bash
 cd ~/electrs
 ```
 
-Update de repository met de laatste wijzigingen via Git.
+Update the repository with the latest changes via Git.
 
 ```bash
 git fetch --all
 ```
 
-Toon de laatste versie/tag/release.
+Show the latest version/tag/release.
 
 ```bash
 git describe --tags `git rev-list --tags --max-count=1`
 ```
 
-Haal de wijzigingen op van de laatste versie.
+Retrieve the changes from the latest release.
 
 ```bash
-git checkout -f <OUTPUT VAN DE VORIGE STAP> # bijvoorbeeld 0.9.7
+git checkout -f <OUTPUT FROM PREVIOUS STEP> # for example 0.9.7
 ```
 
-Installeer de software.
+Install the software.
 
 ```bash
 cargo build --locked --release
 ```
 
-Herstart de Electrs service.
+Restart the Electrs service.
 
 ```bash
 sudo systemctl restart electrs
 ```
 
-Electrs is nu bijgewerkt! Het kan zijn dat tijdens het herstarten - iets dat enkele minuten kan duren - je node niks anders kan doen.
\ No newline at end of file
+Electrs is now updated! You may find that during the restart - something that can take several minutes - your node can't do anything else.
```

Comment:
Heel de file is een soort van dubbel

Reply:
volgens mij fixed, zag het bij 2e run niet meer